http4s-client-monix-catnap/src/main/scala/com/avast/sst/http4s/client/monix/catnap/Http4sClientCircuitBreakerModule.scala

```diff
@@ -0,0 +1,52 @@
+package com.avast.sst.http4s.client.monix.catnap
+
+import cats.effect.{Clock, Resource, Sync}
+import cats.syntax.applicativeError._
+import cats.syntax.flatMap._
+import com.avast.sst.monix.catnap.CircuitBreakerMetrics.State.{Closed, HalfOpen, Open}
+import com.avast.sst.monix.catnap.{CircuitBreakerConfig, CircuitBreakerMetrics, CircuitBreakerModule}
+import org.http4s.Response
+import org.http4s.client.Client
+
+object Http4sClientCircuitBreakerModule {
+
+  /** Wraps [[org.http4s.client.Client]] with [[monix.catnap.CircuitBreaker]] which provides important metrics and logs breaker events.
+    *
+    * The circuit breaker is special in that it also catches any HTTP 5xx responses and considers them as failures for the circuit breaker.
+    */
+  def make[F[_]: Sync](name: String,
+                       client: Client[F],
+                       circuitBreakerConfig: CircuitBreakerConfig,
+                       circuitBreakerMetrics: CircuitBreakerMetrics[F],
+                       clock: Clock[F]): Resource[F, Client[F]] = {
```

Comment:
does this really return resource? Seems like this is only wrapping http client so i don't think there is a reason for return `Resource`

Reply:
:100:

Reply:
Good point.